Hmm interesting. It fails on windows + node 6:

```
C:\Users\travis\build\prebuild\prebuildify\test\package>if not defined npm_config_node_gyp (node "C:\ProgramData\nvs\node\6.16.0\x64\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild --target=v6.16.0 --target_arch=x64 --release )  else (node "" rebuild --target=v6.16.0 --target_arch=x64 --release ) 
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
MSBUILD : error MSB4132: The tools version "2.0" is unrecognized. Available tools versions are "4.0".
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -9,3 +9,6 @@ node_js:
   - 6
   - 8
   - 10
+
+before_install:
+  - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then npm i --global --production windows-build-tools; fi

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,9 +6,5 @@ os:
   - linux
 
 node_js:
-  - 6
   - 8
   - 10
-
-before_install:
-  - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then npm i --global --production windows-build-tools; fi

